### PR TITLE
Base and Solana contract addresses

### PR DESCRIPTION
**Pull request by DLeasure**

### CODE REVIEW

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# V2 Protocol
+
+## Administrative Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| Globals (Proxy) | [`0x7f3C3636208A18c7941BF051807db56864061465`](https://basescan.org/address/0x7f3C3636208A18c7941BF051807db56864061465) | [`globals-v2 @ v2.0.0`](https://github.com/maple-labs/globals-v2/releases/tag/v2.0.0) |
+| MapleTreasury | [`0xa9466EaBd096449d650D5AEB0dD3dA6F52FD0B19`](https://basescan.org/address/0x3A5ab01B4E142F41f54a861c984D19C866Ec1736) | [`maple-core @ v1.0.0`](https://github.com/maple-labs/maple-core/releases/tag/v1.0.0) |
```
Comment (michaelsmueller) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:8]: the  link to `0x3A5ab01B4E142F41f54a861c984D19C866Ec1736` is correct but the listed address is not

Comment (michaelsmueller) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:8]: Why are we linking to `maple-core`? We have a Treasury repo and it seems like you are linking to them, e.g. https://github.com/maple-labs/treasury/releases/tag/v1.0.0

Comment (michaelsmueller) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:8]: I don't think we should link to `maple-core` ... in this case the deployed Treasury is a Gnosis Safe

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| WithdrawalManager | [`0x5B8387aDa310352198EBFF14a297FAb44428C8CD`](https://basescan.org/address/0x5B8387aDa310352198EBFF14a297FAb44428C8CD) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+| WithdrawalManagerInitializer | [`0x899b7d637e35B2C57E845dB1064a4e58639D3A8D`](https://basescan.org/address/0x899b7d637e35B2C57E845dB1064a4e58639D3A8D) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+
+## Factory Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| FixedTermLoanFactory | [`0x1915A8dE08A92b846dF7C845e140E4b0714820bd`](https://basescan.org/address/0x1915A8dE08A92b846dF7C845e140E4b0714820bd) | [`fixed-term-loan @ v2.0.0`](https://github.com/maple-labs/fixed-term-loan/releases/tag/v2.0.0) |
```
Comment (michaelsmueller) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:72]: Farhaan should confirm, but I believe the current release is v3.0.1
https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v3.0.1

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+| WithdrawalManagerFactory | [`0x0434FC9e7D88294dAac40dDF316754B2053D613b`](https://basescan.org/address/0x0434FC9e7D88294dAac40dDF316754B2053D613b) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+
+## Active Pool Contracts
+
+### Cash Management - USDC Pool Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| Pool | [`0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d`](https://basescan.org/address/0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d) | [`pool-v2 @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
+| PoolManager (Proxy) | [`0x7aa13f647691a709dc3360acad7eb758179790ee`](https://basescan.org/address/0x7aa13f647691a709dc3360acad7eb758179790ee) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
```
Comment (michaelsmueller) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:87]: current PoolManager is from v2 release I believe
https://github.com/maple-labs/pool-v2/releases/tag/v2.0.0

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+| FixedTermLoanManager (Proxy) | [`0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491`](https://basescan.org/address/0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
```
Comment (michaelsmueller) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:88]: ditto on FixedTermLoan manager coming from v3 release (not v1) as noted above

Comment (michaelsmueller) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:88]: As noted above, this is a v3 FTLM
https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v3.0.1

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 1 above)
Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:8]: Yup Mike is correct this is one difference from the mainnet protocol, we just use a SAFE as the Treasury

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 2 above)
Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:72]: Appreiciate the confusion here for mainnet we deployed these factories in prior releases but for BASE these factories were deployed with the latest releases from June 2023. 

For FixedTermLoanFactory `fixed-term-loan @ v5.0.1` is the release we deployed 
For FixedTermLoanManagerFactory `fixed-term-loan-manager @ v3.0.1` is the release we deployed
For pool-v2 `pool-v2 @ v2.0.0` is the release we deployed

The other factory versions are correct.

Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:72]: (For reference the factorties are identical to the older versions but for accuracy the versions can be updated as listed above)

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+| WithdrawalManager (Proxy) | [`0x6495f96b89904f574817a3b191c7817d91fe96eb`](https://basescan.org/address/0x6495f96b89904f574817a3b191c7817d91fe96eb) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
+| PoolDelegateCover (Proxy) | [`0x28378DFC7bAe84C668d960c6E8CD862Fda43ba70`](https://basescan.org/address/0x28378DFC7bAe84C668d960c6E8CD862Fda43ba70) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
+
+## Auxiliary Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| FeeManager | [`0xc4D7807a5571887F7b50182d072f4fF4654099b0`](https://basescan.org/address/0xc4D7807a5571887F7b50182d072f4fF4654099b0) | [`fixed-term-loan @ v4.0.0`](https://github.com/maple-labs/fixed-term-loan/releases/tag/v4.0.0) |
```
Comment (0xfarhaan) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:96]: FeeManager would also be `fixed-term-loan @ v5.0.1`

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 3 above)
Comment (0xfarhaan) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:87]: The proxy that is deployed is part of the proxy-factory submodule and here we are linking to the proxy instances hence the user of `proxy-factory @ v1.0.0` so all good.

**Review by 0xfarhaan — COMMENTED** [review]
Left some clarifications but nearly there! Thanks for updating these Dave appreiciate it.

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 1 above)
Comment (DLeasure) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md:8]: Should I take out the whole treasury section?

**Comment by 0xfarhaan** [comment]
Can be shown in the section with the governor and security admin address

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 2 above)
Comment (DLeasure) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md]: Updated

File: technical-resources/protocol-overview/smart-contract-addresses-base.md (hunk 5 above)
Comment (DLeasure) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md]: Updated

File: technical-resources/protocol-overview/smart-contract-addresses-base.md [hunk 6]
```diff
@@ -83,17 +83,17 @@
...
-| Pool | [`0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d`](https://basescan.org/address/0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d) | [`pool-v2 @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
+| Pool | [`0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d`](https://basescan.org/address/0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d) | [`pool-v2 @ v2.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
 | PoolManager (Proxy) | [`0x7aa13f647691a709dc3360acad7eb758179790ee`](https://basescan.org/address/0x7aa13f647691a709dc3360acad7eb758179790ee) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
-| FixedTermLoanManager (Proxy) | [`0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491`](https://basescan.org/address/0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
+| FixedTermLoanManager (Proxy) | [`0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491`](https://basescan.org/address/0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491) | [`fixed-term-loan-manager @ v3.0.1`](https://github.com/maple-labs/fixed-term-loan-manager/commits/v3.0.1) |
```
Comment (0xfarhaan) [inline technical-resources/protocol-overview/smart-contract-addresses-base.md:88]: this should still be proxy-factory @ v1.0.0

Comment (DLeasure) [reply technical-resources/protocol-overview/smart-contract-addresses-base.md]: Ah gotcha, updated

**Review by 0xfarhaan — APPROVED** [review]

**Pull request merged**